request-manager/client.go

```diff
@@ -22,6 +22,10 @@ type Client interface {
 	// GetRequest takes a request id and returns the corresponding request.
 	GetRequest(string) (proto.Request, error)
 
+	// FindRequests takes a request filter and returns a list of requests
+	// matching the filter conditions, in reverse chronological order.
```

Comment:
Reverse chronological = ascending or descending by ts?

Reply:
Descending. I'll reword this a bit to be extra clear.

Reply:
Argh, I was sorting them the wrong way round in the code and managed to write the tests wrong as well 🤦‍♂